javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointCharacteristics.qll

```diff
@@ -143,11 +283,19 @@ private class NosqlInjectionSinkCharacteristic extends EndpointCharacteristic {
  * negative samples for training.
  */
 
+/**
+ * A characteristic that is an indicator of not being a sink of any type, because it's a modeled argument.
+ */
+abstract class OtherModeledArgumentCharacteristic extends EndpointCharacteristic {
```

Comment:
💭: This class defines no logic and it's not used anywhere (other than making some classes descendants). Just for confirmation: I'm assuming the reason you built it is that you want to somehow later use this for class balancing and things of that sort, right?

Reply:
It's used in the definition of `IsArgumentToModeledFunctionCharacteristic`.